webview_flutter/step_12/lib/src/navigation_controls.dart

```diff
@@ -0,0 +1,67 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+class NavigationControls extends StatelessWidget {
+  const NavigationControls({required this.controller, Key? key})
```

Comment:
I would move the key first and add a comma at the end

```suggestion
  const NavigationControls({
    Key? Key,
    required this.controller,
  }) : super(key: key);
```

Reply:
Hilariously we have been putting the Key parameter last. @RedBrogdon do we have any particular feelings on this? We should probably be consistent (and create a lint to enforce the consistency).

Reply:
If I am correct, there currently is a lint to ensure that `child` is always last, but nothing for `key` as far as I  know

Reply:
I changed the order of the parameters in this PR. @domesticmouse  do you want me to make a different PR for the other steps that also have the 'wrong' order of parameters and update that. I can also add the smoke test mentioned by @albertodev01  for all the others step too in that PR.

Reply:
I think the important thing is to be consistent through out the codelab. I don't have a preference one way or the other, as long as it stays stable for the length of the codelab.